jolie/src/main/java/jolie/CommandLineParser.java

```diff
@@ -70,6 +70,7 @@
 	private final int connectionsCache;
 	private final CorrelationEngine.Type correlationAlgorithmType;
 	private final String[] includePaths;
```

Comment:
We have recently modified CommandLineParser to produce an InterpreterParameters object, so this pull request needs to be rebased/merged and modified to account for that. (Should be a small modification.)